build.gradle

```diff
@@ -259,6 +279,25 @@ dependencies {
     rewrite("org.openrewrite.recipe:rewrite-migrate-java")
 }
 
+spotbugs {
+    toolVersion = '4.8.0'
+    ignoreFailures = false
+    showStackTraces = true
+    showProgress = true
```

Comment:
Please change:

```suggestion
    showProgress = false
```

Reason:

![image](https://github.com/JabRef/jabref/assets/1366654/8382781e-5d77-4183-949f-f9a2e8190c1a)

(I hope you can see that the output is VERY large and makes no sense for investigation)